core/src/test/scala/com/yahoo/maha/core/query/BaseQueryChainTest.scala

```diff
@@ -55,6 +56,37 @@ trait BaseQueryChainTest {
     qec
   }
 
```

Comment:
Below function should return no rows from Primary query so that Fallback Query code can be reached.